db/data/20200428152559_add_sector_category_to_activities.rb

```diff
@@ -0,0 +1,17 @@
+class AddSectorCategoryToActivities < ActiveRecord::Migration[6.0]
+  include CodelistHelper
+
+  def up
+    sectors = sector_radio_options
+    activities_with_sectors = Activity.where.not(sector: [nil, ""])
+
+    activities_with_sectors.each do |activity|
+      sector = sectors.find { |sector| sector.code == activity.sector }
```

Comment:
Is it valid for an activity to exist without a `sector_category`? What should happen if one cannot be found? If `activity.update(sector_category: nil)` happens, will it blow up to to validation?